prisondersdilemma.py

```diff
@@ -4,199 +4,438 @@
+		win.flip() 
+		response = waitKeys(keyList=('left', 'right'))
 		deftxt.draw()
 		win.flip()
 		wait(4) #seconds
+		sentencetracker()
+		oppstrategy = 'defect'
 		connecttxt.draw
-		#sentence draw 
 		win.flip()
-	
-	#random coop or defect
-	elif strategy == 3:
+		
+	def random:
+		'''opponent cooperates or defects on a random basis'''
 		shuffle(randostrategy)
-		if randostrategy == 1:
-			opponent.draw()
-			cooptxt.draw()
-			win.flip() 
-			wait(4)#seconds
-			#sentence draw 
-			win.flip()
-		elif: 
-			opponent.draw()
-			deftxt.draw()
-			win.flip()
-			wait(4) #seconds
-			connecttxt.draw
-			#sentence draw 
-			win.flip()
-	
-	elif strategy == 4:
-		if condlist[i] == 1:
-			opponent.draw()
-			cooptxt.draw()
-			win.flip() 
-			wait(4)#seconds
-			connecttxt.draw
-			#sentence draw 
-			win.flip()
-		elif condlist[i] > 1:
-			if resp == 'right':
+		random = randostrategy[0]
+			if random == 1:
 				opponent.draw()
+				participant.draw()
+				controlinfo.draw()
+				win.flip() 
+				response = waitKeys(keyList=('left', 'right'))
 				cooptxt.draw()
+				oppstrategy = 'cooperation'
 				win.flip() 
 				wait(4)#seconds
+				sentencetracker()
 				connecttxt.draw
-				#sentence draw 
 				win.flip()
-			elif resp == 'left':
+			else: 
 				opponent.draw()
-				deftxt.draw()
+				participant.draw()
+				controlinfo.draw()
 				win.flip() 
-				wait(4)#seconds
+				response = waitKeys(keyList=('left', 'right'))
+				deftxt.draw()
+				oppstrategy = 'defect'
+				win.flip()
+				wait(4) #seconds
+				sentencetracker()
 				connecttxt.draw
-				#sentence draw 
 				win.flip()
-	elif strategy == 5:
-		if condlist[i] == 1:
-			opponent.draw()
-			deftxt.draw()
-			win.flip() 
-			wait(4)#seconds
-			connecttxt.draw
-			#sentence draw 
-			win.flip()
-		elif condlist[i] > 1:
-			if resp == 'right':
+				
+	def nicetittat:
+		'''nice tit for that: opponent cooperates in first round, then copies participants moves'''
+		if ntrials[i] == 1:
 				opponent.draw()
+				participant.draw()
+				controlinfo.draw()
+				win.flip() 
+				response = waitKeys(keyList=('left', 'right'))
 				cooptxt.draw()
+				oppstrategy = 'cooperation'
 				win.flip() 
 				wait(4)#seconds
+				sentencetracker()
 				connecttxt.draw
 				#sentence draw 
 				win.flip()
-			elif resp == 'left':
+			else ntrials[i] > 1:
+				if resp == 'right':
+					opponent.draw()
+					participant.draw()
+					controlinfo.draw()
+					win.flip() 
+					response = waitKeys(keyList=('left', 'right'))
+					cooptxt.draw()
+					oppstrategy = 'cooperation'
+					win.flip() 
+					wait(4)#seconds
+					sentencetracker()
+					connecttxt.draw
+					win.flip()
+				else resp == 'left':
+					opponent.draw()
+					participant.draw()
+					controlinfo.draw()
+					win.flip() 
+					response = waitKeys(keyList=('left', 'right'))
+					deftxt.draw()
+					oppstrategy = 'defect'
+					win.flip() 
+					wait(4)#seconds
+					sentencetracker()
+					connecttxt.draw
+					win.flip()
+		
```

Comment:
Is there a way to combine susptittat and nicetittat into a single function? Perhaps utilising arguments going into the function to differentiate between the two? I say this as on first glance, they seem mostly identical.